src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSourceProvider.cs

```diff
@@ -613,6 +616,31 @@ internal void UpdatePackageSource(PackageSource source, bool updateCredentials,
                     isDirty: ref isDirty);
             }
         }
+        private void UpdateAuditSource(
+            PackageSource newSource,
+            PackageSource existingSource,
+            bool shouldSkipSave,
+            ref bool isDirty)
+        {
+            if (string.Equals(newSource.Name, existingSource.Name, StringComparison.OrdinalIgnoreCase))
+            {
+                if ((!string.Equals(newSource.Source, existingSource.Source, StringComparison.OrdinalIgnoreCase) ||
+                    newSource.ProtocolVersion != existingSource.ProtocolVersion ||
```

Comment:
We didn't property strong type this, but protocol version isn't relevant in audit scenarios. 
Audit is only supported by the V3 protocol. I think we should remove it. 

I don't know if we're reading it, but we should stop.

cc @zivkan what do you think?